frontend/packages/console-shared/src/components/form-utils/FormFooter.tsx

```diff
@@ -22,65 +23,74 @@ const FormFooter: React.FC<FormFooterProps> = ({
   hideSubmit = false,
   showAlert,
   sticky,
+  shadow = true,
 }) => {
   const { t } = useTranslation();
+  const [scrollContainer, footerElementRef] = useScrollContainer();
+  const shadowPosition = useScrollShadows(scrollContainer);
```

Comment:
Should find a way to not pass in a `scrollContainer` unless we need to show shadows, otherwise the listeners and calculations are needlessly done.

Reply:
listeners and calculations are only done when `scrollContainer` is available.

Reply:
That's right but before you use to only show shadows when a prop was passed in which made the calculation conditional and therefore should only pass in a non `null` `scrollContainer` if shadows are to be drawn.

BTW I think you still want to not show shadows if the footer isn't sticky? looks like they are always shown now.